apps/js/Question.js

```diff
@@ -158,6 +166,7 @@ export default function Question({
                     onChange={(e) => {
                         setValue(e.target.value);
                     }}
+                    onPaste={submit}
```

Comment:
This may not work, because the `value` state would not have updated in the `submit` closure. You want to write a `handlePaste` that calls `submitValue` with the pasted value.

Reply:
oh how I've missed the javascript event loop..